packages/transport/src/types/apiCall.ts

```diff
@@ -13,6 +13,7 @@ export type ErrorGeneric<ErrorType> = ErrorType extends AnyError
     ? {
           success: false;
           error: ErrorType;
+          message?: string;
```

Comment:
to discuss with @mroz22

Reply:
soo, you found a situation where there was a typed error based on this list https://github.com/trezor/trezor-suite/blob/feat/tranport-api-abortable/packages/transport/src/errors.ts#L1 
and some loosely typed string error message at the same time?

Reply:
hmm yes, this type should be produced either by 
https://github.com/trezor/trezor-suite/blob/feat/tranport-api-abortable/packages/transport/src/utils/result.ts#L12
or by
https://github.com/trezor/trezor-suite/blob/feat/tranport-api-abortable/packages/transport/src/utils/result.ts#L15

and both these functions have error message.

Reply:
maybe something like this? https://github.com/trezor/trezor-suite/pull/13044